qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
`annotation_help_text` is confusing to me because the annotations (in the context of QIIME 2) are Python types, but the function inputs and outputs are being documented here, not their annotations. Maybe `signature_descriptions`, `signature_docs`, or `signature_help_text` is clearer? I don't really like any of those but can't think of a better name.

Another option is to have three description dictionaries: `input_descriptions`, `parameter_descriptions`, `output_descriptions`. Then it's pretty clear what these descriptions apply to.

(Note: I'm favoring "description" over "help text" because we've used `description` in `register_function` to describe the method/visualizer).